weblate/wladmin/forms.py

```diff
@@ -59,29 +60,52 @@ def __init__(self, **kwargs) -> None:
         )
 
 
-class ColorField(forms.CharField):
-    def __init__(self, **kwargs) -> None:
-        super().__init__(widget=forms.TextInput(attrs={"type": "color"}), **kwargs)
+class ThemeColorWidget(MultiWidget):
+    def __init__(self, attrs=None):
+        widgets = (
+            forms.TextInput(attrs={"type": "color", "class": "light-theme"}),
+            forms.TextInput(attrs={"type": "color", "class": "dark-theme"}),
+        )
+        super().__init__(widgets, attrs)
+
+    def decompress(self, value):
+        if value:
+            return value.split(",")
+        return [None, None]
+
+
+class ThemeColorField(forms.MultiValueField):
+    widget = ThemeColorWidget
+
+    def __init__(self, **kwargs):
+        fields = (forms.CharField(required=False), forms.CharField(required=False))
+        super().__init__(fields=fields, require_all_fields=False, **kwargs)
+
+    def compress(self, data_list):
+        return ",".join(data_list) if data_list else None
```

Comment:
Can't we just store the list in the database field? That would avoid the need for all the split/join logic.

Reply:
Tbh I'm not sure what is the best choice in this case. What you see is better we will go with it

Reply:
I'd prefer to avoid conversion to string, it's backed by a JSON field in the end, so it should have issues with storing list.